src/services/api/index.ts

```diff
@@ -0,0 +1,8 @@
+import axios from 'axios'
+
+export const api = axios.create({
+  baseURL: `http://localhost:4000`,
```

Comment:
This should most def be part of a .env file. I suggest adding a .env.example file which is explained in the README.

Reply:
Yeah, true! Completely agree and will be adding it in the next commit, tho. 

But just to clear out here, you can see in the past commits probably, it had the .env with the proper data as I was planning to put it up on Heroku, but then I changed my mind and thought the .env was unnecessary in such a small project with only one environment variable, lol.